EMS-UI/angular.json

```diff
@@ -41,10 +41,10 @@
               "sourceMap": false,
               "extractCss": true,
               "namedChunks": false,
-              "aot": true,
+              "aot": false,
```

Comment:
can these be switched back on or still break?

Reply:
Unfortunately `aot` and `buildOptimizer` still break the runtime, however I realised that when we were doing `ng build` as opposed to `ng build --prod` we weren't actually getting any of those other nice angular options such as `sourceMap: false`. Now we've switched to building with `ng build --prod` and only missing out on those 2 flags.